README.md

```diff
@@ -21,19 +22,22 @@
  - [Authors](#authors)
 
 ## Installing
-Copy the [io_ogre](io_ogre) folder into the [$BLENDER_DIR](https://docs.blender.org/manual/en/latest/advanced/blender_directory_layout.html)`/scripts/addons` folder.
-Note on Windows: you have to set the correct path to `OGRETOOLS_XML_CONVERTER` in [io_ogre/config.py](io_ogre/config.py) prior to the first run.  
-After installing the addon enable it in Blender from `User Preferences > Add-Ons > Import-Export` (CTRL+ALT+U). Search for `ogre` and check the box on the right. 
-Remember to save as default if you want the addon to be enabled after you exit your Blender.  
-Integrated help docs will be shown in the upper right hand toolbar, replacing blender's normal `Help` menu, read them for assistance.
+1. Copy the [io_ogre](io_ogre) folder into the [$BLENDER_DIR](https://docs.blender.org/manual/en/latest/advanced/blender_directory_layout.html)`/scripts/addons` folder.
+  i. Windows = %USERPROFILE%\AppData\Roaming\Blender Foundation\Blender\[VERSION]\scripts\addons
 
-The following versions of blender have been tested.
+2. Set the correct path to `OGRETOOLS_XML_CONVERTER` in [io_ogre/config.py](io_ogre/config.py) (Line 108) prior to the first run.
+   i. If you want to export meshes for Ogre (v1): Then this path should point to `OgreXMLConverter.exe`. This file can be found in the [Ogre SDK](https://www.ogre3d.org/download/sdk/sdk-ogre)
+  ii. If you want to export meshes for OgreNext (v2.*): Then this path should point to `OgreMeshTool.exe`. This file can be found in the [OgreNext SDK](https://www.ogre3d.org/download/sdk/sdk-ogre-next)
 
+3. Enable the addon in Blender
+   i. Version 2.9: `Edit menu > Preferences > Add-ons`. Search for `ogre` and click the box up the top left.
+  ii. Version 2.7: `User Preferences > Add-Ons > Import-Export` (CTRL+ALT+U). Search for `ogre` and check the box on the right. Remember to save as default if you want the addon to be enabled after you exit your Blender. Integrated help docs will be shown in the upper right hand toolbar, replacing blender's normal `Help` menu, read them for assistance.
```

Comment:
2.7 is handled in a separate branch

Reply:
Ha. I was guessing the Blender version. I only have 2.93 and the instructions were different to what I was looking at so was updating them to be more current. I took a stab that the instruction were good for 2.7 but maybe those instructions are for 2.8 or from a long time ago?

Is it best to just remove the instrucitons for what I said was 2.7? Are new, less knowledgable people more likely to use an up to date version of Blender that they just downloaded?

Reply:
those instructions seem to even predate 2.7. Lets focus on 2.8+ here (the ones you added). As everything is 2.8+ in master, no need to mention a version.